Update minimal requirements (#901)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-Introduction.md

:::details
### What are the minimal requirements to run Wasabi?

Wasabi runs in most operating systems with 64-bit architecture.

> For the complete list of all the officially supported software and devices by Wasabi Wallet, click [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/WasabiCompatibility.md).

Reviewer: ```suggestion
For the complete list of all the officially supported operating systems, click [here](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/WasabiCompatibility.md#officially-supported-operating-systems).
```
